web/public/index.html

```diff
@@ -1,44 +1,84 @@
-        <link rel="preload" href="/fonts/OpenSans-Bold.woff2" as="font" type="font/woff2" crossorigin>
-        <link rel="preload" href="/fonts/EuclidTriangle-Regular.woff" as="font" type="font/woff" crossorigin>
-
-        <!-- Set Global Variables -->
-        <script>
-            var ELECTRICITYMAP_PUBLIC_TOKEN = '%SNOWPACK_PUBLIC_ELECTRICITYMAP_PUBLIC_TOKEN%';
-            var bundleHash = 'dev';
-            var resolvePath = relativePath => `/${relativePath}`;
-            var isCordova = false;
-        </script>
-    </head>
-    <body id="app"></body>
-</html>
\ No newline at end of file
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:fb="http://ogp.me/ns/fb#" lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="description"
+      content="electricityMap is a live visualization of where your electricity comes from and how much CO2 was emitted to produce it."
+    />
+    <meta name="viewport" content="width=device-width, viewport-fit=cover" />
+
+    <meta property="og:locale" content="en" />
+
+    <meta
+      http-equiv="Content-Security-Policy"
+      content="default-src * 'self' 'unsafe-inline' data: gap: https://ssl.gstatic.com 'unsafe-eval' blob:; style-src * 'unsafe-inline'; media-src *;"
+    />
+
+    <meta name="apple-itunes-app" content="app-id=1224594248" />
+
+    <title>electricityMap | dev</title>
+
+    <link rel="icon" type="image/svg+xml" href="/images/electricitymap-favicon-dark-light.svg" />
+    <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-icon-180x180.png" />
+    <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-32x32.png" />
+    <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16.png" />
+    <link rel="manifest" href="/manifest.json" />
+    <meta name="theme-color" content="#ffffff" />
+
+    <!-- Pre-connect to improve performances -->
+    <link rel="preconnect" href="https://app-backend.electricitymap.org/" />
+
+    <!-- Bundle -->
+    <script type="module" src="/dist/index.js"></script>
+
+    <!-- Preload font files to improve performance-->
+    <link rel="preload" href="/fonts/OpenSans-Regular.woff2" as="font" type="font/woff2" crossorigin />
+    <link rel="preload" href="/fonts/OpenSans-Bold.woff2" as="font" type="font/woff2" crossorigin />
+    <link rel="preload" href="/fonts/EuclidTriangle-Regular.woff" as="font" type="font/woff" crossorigin />
+
+    <!-- Set Global Variables -->
+    <script>
+      var ELECTRICITYMAP_PUBLIC_TOKEN = '%SNOWPACK_PUBLIC_ELECTRICITYMAP_PUBLIC_TOKEN%';
+      var bundleHash = 'dev';
+      var resolvePath = (relativePath) => `/${relativePath}`;
+      var isCordova = false;
+    </script>
+    <style>
+      html,
+      body {
+        margin: 0 !important;
+        padding: 0 !important;
+        font-family: 'Open Sans', sans-serif;
+        transition: background-color 800ms;
+        overflow: hidden; /* Disable dragging that can occur on mobile */
+      }
```

Comment:
I don't think we need to implement this styling directly here, in the build it's loading CSS fine with JS disabled :)

<img width="555" alt="Screenshot 2022-05-23 at 13 47 20" src="https://user-images.githubusercontent.com/3296643/169812667-5cc8f5f6-e121-4fd7-bceb-a1f4bafdc7a1.png">
<img width="1112" alt="Screenshot 2022-05-23 at 13 47 12" src="https://user-images.githubusercontent.com/3296643/169812677-31135e14-6c90-4a95-8910-1e2decc49d23.png">

Reply:
Hmm it didn't when I tried it last time.

I'm not at my computer right so can't check right now but I'll test it on Edge and Firefox when I get back. (might be browser specific)

Reply:
It makes sense if it fails during local development, but it should be the same on production no matter the browser? 🤔

Reply:
Found the problem! We don't actually load the CSS bundles in html when using snowpack but rather snowpack use js to inject them (for HMR?).

I'll change back the ejs template when I get back and keep the styles in the snowpack index file when I get back.